testsuite/MDAnalysisTests/analysis/test_diffusionmap.py

```diff
@@ -80,3 +80,16 @@ def test_transform(u, dmap):
     dmap.run()
     diffusion_space = dmap.transform(n_eigenvectors, 1)
     assert diffusion_space.shape == (eigvects.shape[0], n_eigenvectors)
+
+def test_long_traj(u, tmpdir):
```

Comment:
Is the `tmpdir` fixture actually used for something here? I don't see it being used?

If there is an artifact being produced, I believe `with tmpdir.as_cwd():` is a popular choice with the core team.

Reply:
Ugh, no sorry. I forgot to erase that from the trial I ran using a dummy trajectory with 5000 frames. The test run with `u` only.